### ISSUE TITLE
Keyboard support

### ISSUE THREAD

**Issue by MrEricSir** [MrEricSir/QMLRearrangeableTreeView#5]
There should be a way to move items around via the keyboard.  This would be useful for accessibility purposes and for automated testing.
